shiksha-migration/cohort-academicYear-userdb-migration.js

```
@@ -0,0 +1,187 @@
+      total: cohorts.length,
+      inserted,
+      skipped,
+      failed,
+    });
+
+    console.log(
+      '[COHORT_ACADEMIC_YEAR] Migration completed successfully'
+    );
+
+  } catch (err) {
+
```

ESLint

[error] Delete `⏎`

(prettier/prettier)